src/pages/Proposal/ProposalSummary/index.tsx

```diff
@@ -125,6 +138,7 @@ export const ProposalSummaryUi: React.FC<
           onClick={handleExecuteProposal}
           css={styles.updateProposalButton}
           loading={isExecuteProposalLoading}
+          data-testid={TEST_IDS.proposalSummary.executeButton}
```

Comment:
In this instance we can use the content directly to target the button since it doesn't include any string interpolation

Reply:
This is a valid solution. It is better than using the translation string because the text appears multiple times on the page. Using a testId allows the selection of the specific component we want to use